src/Nethermind/Nethermind.Merge.Plugin/BlockProduction/PayloadPreparationService.cs

```diff
@@ -175,6 +175,7 @@ private void CleanupOldPayloads(object? sender, EventArgs e)
     private void LogProductionResult(Task<Block?> t, Block currentBestBlock, UInt256 blockFees, TimeSpan time)
     {
         const long weiToEth = 1_000_000_000_000_000_000;
+        const long weiToGwei = 1_000_000_000;
```

Comment:
Don't we have a shared denomination elsewhere?

Reply:
UInt256 based